### DISCUSSION TITLE
Grid causing scrollbar on smaller screen

### DISCUSSION THREAD

**Discussion by pixelize** [tailwindlabs/tailwindcss#6682] [Help] [answered]
Hi everyone,

If the screen is smaller (like on mobile) the grid cause a horizontal scroll.

What I'am doing wrong ? Is it possible to fix this using a 12 cols grid all the time like a regular grid system ? 

https://play.tailwindcss.com/HS8VbJ3fXt

**Comment by cenkakay** [comment, marked as answer]
Have you ever tried to change gap property?
Or maybe you can use overflow-clip?

**Reply by pixelize** [reply]
Hi @cenkakay reducing gap works but i'd like to keep the same gap on smaller screen as well.

`overflow: clip` seems to work on the grid container ! Thanks a lot

**Reply by cenkakay** [reply]
Good hear that. You can use overflow hidden instead of clip. Safari dosent support clip.
